css/style.css

```diff
@@ -128,6 +126,99 @@ a {
 	text-indent: -1.5em;
 }
 
+#ideasContainer {
+	background: transparent;
+	padding: 4em 5.8em;
+	margin: 0 auto;
+}
+
+.ideas h2 {
+	font-family: 'Comfortaa', sans-serif;
+	color: #e7e7e7;
+	padding: 0 0;
+	margin: 0;
+	font-size: 1.5em;
+	text-align: center;
+}
+
+.ideas p {
+	font-family: 'Fira Sans', sans-serif;
+	font-weight: 300;
+	color: #e7e7e7;
+	margin: 0.8em 0;
+	font-size: 1.2em;
+	line-height: 1.6em;
+	letter-spacing: 0.04em;
+}
+
+/* Timeline
+	Brady Wright https://codepen.io/brady_wright/pen/NNOvrW
+	Niels Voogt https://codepen.io/NielsVoogt/pen/MbMMxv/
```

Comment:
How much content from others are we using? Do we know about licensing?

Reply:
From a note in the channel: all [source code posted in public CodePens are licensed under the MIT license](https://blog.codepen.io/legal/licensing/).